From 3a23b3a3260c11e68c65d1dea6636d1e7fcd8a7e Mon Sep 17 00:00:00 2001
From: Yann Gautier <yann.gautier@st.com>
Date: Fri, 22 Sep 2023 09:51:06 +0200
Subject: [PATCH 253/841] feat(stm32mp2-fdts): enable HASH, PKA, RNG and SAES
 for eval board

Add status = "okay" for hash, pka, rng and saes nodes for STM32MP257F
eval board. Those peripherals are required for Trusted Boot.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/arm-trusted-firmware.git c3a57038b5499f4b54314a3cb3041d51ee70fd51 feat(stm32mp2-fdts): enable HASH, PKA, RNG and SAES for eval board]
Signed-off-by: Yann Gautier <yann.gautier@foss.st.com>
Change-Id: Ie29fd1940d5019a325bf9c1d3f079c4c49708a5a
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/tf-a/+/285749
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Domain-Review: Lionel DEBIEVE <lionel.debieve@foss.st.com>

# Conflicts:
#	fdts/stm32mp257f-ev1.dts
---
 fdts/stm32mp257f-ev1.dts | 16 +++++++++++++---
 1 file changed, 13 insertions(+), 3 deletions(-)

diff --git a/fdts/stm32mp257f-ev1.dts b/fdts/stm32mp257f-ev1.dts
index 8b4648f3e..19d3bf444 100644
--- a/fdts/stm32mp257f-ev1.dts
+++ b/fdts/stm32mp257f-ev1.dts
@@ -40,9 +40,7 @@
 	};
 };
 
-&usart2 {
-	pinctrl-names = "default";
-	pinctrl-0 = <&usart2_pins_a>;
+&hash {
 	status = "okay";
 };
 
@@ -67,6 +65,18 @@
 	};
 };
 
+&pka {
+	status = "okay";
+};
+
+&rng {
+	status = "okay";
+};
+
+&saes {
+	status = "okay";
+};
+
 &sdmmc1 {
 	pinctrl-names = "default";
 	pinctrl-0 = <&sdmmc1_b4_pins_a>;
-- 
2.47.2

